{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2020-35381",
      "description": "jsonparser 1.0.0 allows attackers to cause a denial of service (panic: runtime error: slice bounds out of range) via a GET call.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-35381",
      "reportURL": "https://github.com/buger/jsonparser/issues/219",
      "timeStamp": "2021-01-17T04:15:00Z"
    }
  ],
  "trickest": [],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2020-35381",
      "url": "https://github.com/buger/jsonparser/issues/219",
      "added_at": "2021-01-17T04:15:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
